Gems/Atom/Tools/ShaderManagementConsole/Code/CMakeLists.txt

```diff
@@ -95,7 +92,6 @@ ly_add_target_dependencies(
     TARGETS
         ShaderManagementConsole
     DEPENDENCIES_FILES
-        tool_dependencies.cmake
```

Comment:
The platform specific `tool_dependencies_<platform>.cmake` can be instead be moved to a `PLATFORM_INCLUDE_FILES`  argument as part of the `ShaderManagementConsole` target,

The `LY_RUNTIME_DEPENDENCIES` is just the variable that needs to be set in that file.

Reply:
i.e instead of `set(GEM_DEPENDENCIES` it can be changed to `set(LY_RUNTIME_DEPENDENCIES`